src/Sylius/Component/Core/Dashboard/SalesDataProvider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Dashboard;
```

Comment:
I would consider putting it to the admin bundle

Reply:
I would consider it also, but we can do it later, as this class is already experimental 🥄